### PR TITLE
Feature/hs 1881

### PR DESCRIPTION
**Pull request by dv-Gogna**
Some code blocks have extra spaces around the actual content.
so we have removed those extra spaces.

### CODE REVIEW

File: docs/android/tutorials.mdx [hunk 1]
```diff
@@ -102,7 +102,7 @@ on Android Developers.
 Copy the image files for the tiled background to your `ldpi`, `mdpi`, `hdpi`
 and `xhdpi` drawable folders.
 
-```
+```xml
```
Comment (hs-hritik) [inline docs/android/tutorials.mdx:101]: Replaace `xml` with `text`

File: docs/ios/delegates.mdx [hunk 2]
```diff
@@ -288,7 +277,6 @@ This delegate method is called when user clicks on action card button.
 Example usage:
 
 ```objc
-
```
Comment (hs-hritik) [inline docs/ios/delegates.mdx:291]: `}` is out of the code blocks, could add that inside the code blocks.

**Review by hs-hritik — CHANGES_REQUESTED** [review]
LGTM. Added some minor suggestion. Thanks for doing this.

File: docs/cocos2d-x/sdk-configuration-android.mdx [hunk 3]
```diff
@@ -702,7 +680,6 @@ Example:
 <TabItem value="For Cocos2d-x 3.x " label="For Cocos2d-x 3.x ">
 
 ```cppp
```
Comment (hs-saunved) [inline docs/cocos2d-x/sdk-configuration-android.mdx:657]: This should be `cpp`
Please check if there are other such instances where the code block value is incorrect.

File: docs/android/troubleshooting.mdx [hunk 4]
```diff
@@ -101,7 +101,7 @@ Once done you can run a `clean & compile`. As these classes are already included
 
 If you get a dex merger error like below, enable the jumbo mode.
 
-```
+```java
```
Comment (hs-saunved) [inline docs/android/troubleshooting.mdx:102]: This is not Java.
Please make this `text`

File: docs/android/troubleshooting.mdx [hunk 5]
```diff
@@ -135,7 +135,7 @@ To enable jumbo mode in Ant add the following to your project.properties -
 
 If you get a dex error like below,
 
-```
+```java
```
Comment (hs-saunved) [inline docs/android/troubleshooting.mdx:128]: Same as above.

File: docs/android/troubleshooting.mdx [hunk 6]
```diff
@@ -158,7 +158,7 @@ You can resolve this by deleting the **BuildConfig** file which comes with the H
 
 If you get a crash with the following error, check the android support library version you use:
 
-```
+```java
```
Comment (hs-saunved) [inline docs/android/troubleshooting.mdx:147]: Same as above

File: docs/android/troubleshooting.mdx [hunk 7]
```diff
@@ -217,7 +217,7 @@ Application.onCreate() {
 
 ### Helpshift Logs with low priority DEBUG
 
-```
+```java
```
Comment (hs-saunved) [inline docs/android/troubleshooting.mdx:203]: Same as above

File: docs/android/troubleshooting.mdx [hunk 8]
```diff
@@ -229,7 +229,7 @@ You can safely ignore the Helpshift SDK debug logs (log priority "DEBUG" / "Log.
 
 ### Helpshift Warning with low priority
 
-```
+```java
```
Comment (hs-saunved) [inline docs/android/troubleshooting.mdx:213]: Same as above

File: docs/pc-and-console/helpshift-apis.mdx [hunk 9]
```diff
@@ -385,7 +385,7 @@ When there is an open issue for the end user -
 
 When there are no open issues for the end user -
 
-```
+```json
```
Comment (hs-saunved) [inline docs/pc-and-console/helpshift-apis.mdx:388]: Same as above

File: docs/web/helpshift-apis.mdx [hunk 10]
```diff
@@ -343,7 +343,7 @@ When there is an open issue for the end user -
 
 When there are no open issues for the end user -
 
-```
+```json
```
Comment (hs-saunved) [inline docs/web/helpshift-apis.mdx:346]: text

**Review by hs-saunved — CHANGES_REQUESTED** [review]

**Review by hs-hritik — APPROVED** [review]

File: docs/ios/delegates.mdx (hunk 2 above)
Comment (hs-hritik) [reply docs/ios/delegates.mdx:291]: @dv-Gogna I think, this is not resolved.

**Review by hs-saunved — APPROVED** [review]

**Pull request merged**